Testscripts/Linux/utils.sh

```diff
@@ -2209,11 +2209,8 @@ function install_sshpass () {
 function add_sles_benchmark_repo () {
 	if [ $DISTRO_NAME == "sles" ]; then
 		case $DISTRO_VERSION in
-			11*)
-				repo_url="https://download.opensuse.org/repositories/benchmark/SLE_11_SP4/benchmark.repo"
-				;;
 			12*)
-				repo_url="https://download.opensuse.org/repositories/benchmark/SLE_12_SP4/benchmark.repo"
+				repo_url="https://download.opensuse.org/repositories/benchmark/SLE_12_SP5/benchmark.repo"
```

Comment:
Can we get that number from the original string parsing and add it into the url?